### PR TITLE
cilium: add visibility for all flags in CT dump

### PR DESCRIPTION
**Pull request by borkmann**
While debugging #10942, I noticed that not all flags are currently
dumped in our bugtool report's CT list. Fix it by adding the missing
ones as well.

Signed-off-by: Daniel Borkmann <daniel@iogearbox.net>

### CODE REVIEW

**Comment by borkmann** [comment]
test-me-please

**Review by brb — APPROVED** [review]

File: pkg/maps/ctmap/types.go [hunk 1]
```diff
@@ -487,12 +487,14 @@ type CtEntry struct {
...
-	SeenNonSyn = 1 << 4
-	NodePort   = 1 << 5
+	RxClosing     = 1 << 0
+	TxClosing     = 1 << 1
+	Nat64         = 1 << 2
+	LBLoopback    = 1 << 3
+	SeenNonSyn    = 1 << 4
+	NodePort      = 1 << 5
+	ProxyRedirect = 1 << 6
+	DSR           = 1 << 7
```
Comment (tklauser) [inline pkg/maps/ctmap/types.go]: Nit: could use `iota` to define this and make future additions easier:

```suggestion
	RxClosing     = 1 << iota
	TxClosing
	Nat64
	LBLoopback
	SeenNonSyn
	NodePort
	ProxyRedirect
	DSR
```

Comment (borkmann) [reply pkg/maps/ctmap/types.go]: Ah yeah, good point, will change it into this style.

**Review by qmonnet — APPROVED** [review]

**Review by joestringer — COMMENTED** [review]
For future proofing, we should probably consider matching on any currently-unknown flags and printing their hex value so that we actually know that we're missing the translation for them but we will actually have the value :-)

**Review by aanm — APPROVED** [review]
LGTM once Tobias comment is solved.

**Comment by borkmann** [comment]
test-me-please

**Review by tklauser — APPROVED** [review]

**Pull request merged**